test/e2e/operator_test.go

```diff
@@ -506,7 +506,13 @@ func TestDNSOverTLSForwarding(t *testing.T) {
 	// Create a separate namespace for the upstream resolver. Deleting this namespace will clean up the resources created.
 	tlsUpstreamNamespace := &corev1.Namespace{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: "openshift-testdnsovertls",
+			Name: tlsUpstreamName,
+			Labels: map[string]string{
+				"pod-security.kubernetes.io/warn":                "privileged",
+				"pod-security.kubernetes.io/audit":               "privileged",
+				"pod-security.kubernetes.io/enforce":             "privileged",
```

Comment:
So basically all of these labels are creating a privileged namespace?

Reply:
Correct.